symplyphysics/laws/kinematic/vector/acceleration_relative_to_reference_frame.py

```diff
@@ -0,0 +1,87 @@
+from sympy import Expr, symbols
+from symplyphysics import (
+    units,
+    validate_input,
+    validate_output,
+    Vector,
+    Quantity,
+    QuantityVector,
+    add_cartesian_vectors,
+    subtract_cartesian_vectors,
+    scale_vector,
+)
+from symplyphysics.core.vectors.arithmetics import (
+    diff_cartesian_vector,
+    integrate_cartesian_vector,
+)
+
+# Description
+## For any reference frame, whether it is inertial or not, the motion relative to it can be described using
+## the position vector relative to that frame's origin.
+
+# Law: a_rel = (d**2(x)/dt**2, d**2(y)/dt**2, d**2(z)/dt**2)
+## a_rel - vector of acceleration relative to reference frame S
+## r = (x, y, z) - vector of position relative to frame S
+## d**2/dt**2 - second-order derivative w.r.t. time
+
+def relative_acceleration_law(
+    position_: Vector,
+    time_: Expr,
+) -> Vector:
+    return diff_cartesian_vector(position_, time_, time_)
+
+
+# v = v0 + Integral(a(t), t)
+def relative_velocity_law(
```

Comment:
Why velocity is here? Our parameters are position and acceleration.

Reply:
Okay, I guess it's maybe better to formulate this law as acceleration being time derivative of velocity.